comp/logs/agent/config/config.go

```diff
@@ -17,6 +17,7 @@ import (
 	pkgConfig "github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/config/utils"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	pointer "github.com/DataDog/datadog-agent/pkg/util/pointer"
```

Comment:
is it necessary to have this alias since the filename ends with `pointer` already?